tests/parametric/test_telemetry.py

```diff
@@ -148,23 +148,24 @@ def test_library_settings(self, library_env, test_agent, test_library):
         configuration = event["payload"]["configuration"]
         configuration_by_name = {item["name"]: item for item in configuration}
 
-        # Check that the tags name match the expected value
+        # # Check that the tags name match the expected value
         assert configuration_by_name.get("DD_ENV").get("value") == "dev"
         assert configuration_by_name.get("DD_SERVICE").get("value") == "service_test"
         assert configuration_by_name.get("DD_VERSION").get("value") == "5.2.0"
-        assert configuration_by_name.get("DD_TRACE_RATE_LIMIT").get("value") == 10
+        assert configuration_by_name.get("DD_TRACE_RATE_LIMIT").get("value") == "10"
         assert (
             configuration_by_name.get("DD_TRACE_HEADER_TAGS").get("value") == "User-Agent:my-user-agent,Content-Type."
         )
-        assert configuration_by_name.get("DD_TRACE_ENABLED").get("value") == False
+        assert configuration_by_name.get("DD_TRACE_ENABLED").get("value") == True
         assert configuration_by_name.get("DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP").get("value") == "\d{3}-\d{2}-\d{4}"
-        assert configuration_by_name.get("DD_TRACE_LOG_DIRECTORY").get("value") == "/some/temporary/directory"
+        if context.library != "nodejs":
```

Comment:
Could you add a comment explaining why is it different in node ?

Reply:
> these settings are not supported by the nodejs sdk

Could you split the test method in two separated test method, and decorate the one with the unsupported part with `@missing_feature` ?